httpx/_content_streams.py

```diff
@@ -276,7 +276,7 @@ def __init__(self, data: dict, files: dict, boundary: bytes = None) -> None:
         body = BytesIO()
         if boundary is None:
             boundary = binascii.hexlify(os.urandom(16))
-
+            
```

Comment:
Careful:

```suggestion

```

(This made CI fail, you can run code formatting locally with `$ scripts/lint`.)